python/felis/datamodel.py

```diff
@@ -83,40 +91,83 @@ class BaseObject(BaseModel):
     """
 
     description: DescriptionStr | None = None
-    """A description of the database object.
-
-    By default, the description is optional but will be required if
-    `BaseObject.Config.require_description` is set to `True` by the user.
-    """
+    """A description of the database object."""
 
-    @model_validator(mode="before")
+    @model_validator(mode="after")  # type: ignore[arg-type]
```

Comment:
I think I see that you already figured this out on Slack, but if you drop the `@classmethod` I think you won't need the `type: ignore` anymore.